recipes/jupyter_sphinx/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "jupyter_sphinx" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "3103d4e905b63f6f94b991824e47130ab9c0bdc94fe427fa8090a1e646498787" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - sphinx
+    - ipywidgets >=6.0.0
+
+test:
+  imports:
+    - jupyter_sphinx
+
+about:
+  home: https://github.com/jupyter-widgets/jupyter-sphinx
+  license: BSD 3-Clause
+  license_family: BSD
+  # license_file: LICENSE
```

Comment:
Would be a great if we could get a PR going to fix issue ( https://github.com/jupyter-widgets/jupyter-sphinx/issues/8 ).